instance/models/mixins/openedx_theme.py

```diff
@@ -71,6 +72,13 @@ def get_simple_theme_static_files_settings(self, application):
             },
         ]
 
+        if application.hero_cover_image:
```

Comment:
@lgp171188 Did you get a chance to test if this change is properly deployed?

Reply:
I'm worried that the file might not be deployed to the appserver.

Reply:
@giovannicimolin, this uses the same logic as the favicon and logo images. So I see no reason why this shouldn't work.